### PR TITLE
Kinetic panda move group py

### PR DESCRIPTION
**Pull request by mlautman**
This replaces [#14](https://github.com/PickNikRobotics/moveit_tutorials/pull/14)

### CODE REVIEW

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py [hunk 1]
```diff
@@ -83,9 +84,8 @@ def move_group_python_interface_tutorial():
                                       moveit_msgs.msg.DisplayTrajectory,
                                       queue_size=20)
 
-  ## Wait for RVIZ to initialize. This sleep is ONLY to allow Rviz to come up.
-  print "============ Waiting for RVIZ..."
-  rospy.sleep(10)
+  print "============ Press Enter to Begin the Tutorial..."
+  raw_input()
```
Comment (davetcoleman) [inline doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: love it

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py [hunk 2]
```diff
@@ -110,27 +110,44 @@ def move_group_python_interface_tutorial():
...
+  ## Note that we are just planning, not asking move_group
   ## to actually move the robot
   plan1 = group.plan()
-  
 
-  print "============ Waiting while RVIZ displays plan1..."
-  rospy.sleep(5)
 
- 
+  print "============ Press Enter to Continue the Tutorial After RVIZ Displays Plan1..."
```
Comment (davetcoleman) [inline doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: Rviz

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py [hunk 3]
```diff
@@ -141,32 +158,32 @@ def move_group_python_interface_tutorial():
   display_trajectory.trajectory.append(plan1)
   display_trajectory_publisher.publish(display_trajectory);
 
-  print "============ Waiting while plan1 is visualized (again)..."
-  rospy.sleep(5)
+  print "============ Press Enter to Continue the Tutorial After RVIZ Displays Plan1 (again)..."
```
Comment (davetcoleman) [inline doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: Rviz through out doc

**Comment by mlautman** [comment]
Check out the fancy [live version](http://tutorials.picknik.ai/doc/move_group_python_interface/move_group_python_interface_tutorial.html)

File: doc/move_group_python_interface/move_group_python_interface_tutorial.rst [hunk 4]
```diff
@@ -7,39 +7,47 @@ specifically setting joint or pose goals, creating motion plans, moving the
...
 the robot.
 
-.. tutorial-formatter:: ./scripts/move_group_python_interface_tutorial.py
-
 The entire code
-^^^^^^^^^^^^^^^
+---------------
 The entire code can be seen :codedir:`here in the MoveIt! Github project<move_group_python_interface/scripts/move_group_python_interface_tutorial.py>`.
 
+.. tutorial-formatter:: ./scripts/move_group_python_interface_tutorial.py
```
Comment (davetcoleman) [inline doc/move_group_python_interface/move_group_python_interface_tutorial.rst]: move this to the bottom

**Pull request merged**